portfolio/src/main/webapp/script.js

```diff
@@ -51,6 +58,10 @@ function loadGallery(){
 
 }
 
+function showComments(comments){
```

Comment:
Either delete this function from this pull request and include it in the next one once it's finished, or finish implementing it before submitting this pull request.